gulpfile.js

```diff
@@ -32,6 +32,7 @@ gulp.task('import-docs', function (cb) {
     if (err instanceof Error) {
       cb(err);
     }
+    //console.log(stdout);
```

Comment:
I purposely kept it commented out because I'm always forgetting how to turn it on.  I thought it no harm to leave it in, but commented.